arangodb-net-standard.Test/ApiClientTestFixtureBase.cs

```diff
@@ -60,7 +60,10 @@ protected async Task CreateDatabase(string dbName)
                 }
                 finally
                 {
-                    _databases.Add(dbName);
+                    if(exclude != true)
```

Comment:
I think this would be clearer as `if (!exclude)`

Reply:
Actually I don't think we should do this. I want to track all databases we create so we delete them when the fixture `Dispose` method is called.  We already have try/catch there so even if the database was already deleted it should not cause a problem.  I don't want there to be any databases left behind accidentally if the test fails.